constrained_linear_regression/__init__.py

```diff
@@ -44,7 +44,13 @@ def fit(self, X, y, min_coef=None, max_coef=None):
         if self.nonnegative:
             self.min_coef_ = np.clip(self.min_coef_, 0, None)
 
-        beta = np.zeros(X.shape[1]).astype(float)
+        if initial_beta is not None:
+            # providing initial_beta may be useful,
+            # if initial solution does not respect the constraints.
+            beta = initial_beta
```

Comment:
We should verify that `initial_beta` has the right shape; otherwise, we may end up having weird errors.

Reply:
I introduce shape check and test for initial_beta and min and max coef. [9fb6c79](https://github.com/avidale/constrained-linear-regression/pull/7/commits/9fb6c79d08b7bc731862093fd679fc7dc9913247)